packages/preferences/src/browser/util/preference-tree-generator.ts

```diff
@@ -94,24 +56,29 @@ export class PreferenceTreeGenerator {
         const groups = new Map<string, Preference.CompositeTreeNode>();
         const root = this.createRootNode();
 
-        for (const id of this.topLevelCategories.keys()) {
-            this.getOrCreatePreferencesGroup(id, id, root, groups);
+        const commonlyUsedLayout = this.layoutProvider.getCommonlyUsedLayout();
```

Comment:
It messes with the algorithm that determines to which group a preference actually belongs. I've tried that, at it basically rendered those preferences **only** in the commonly used group - and nowhere else. It's supposed to render the preference twice in case it's in the commonly used group, which is why it needs special handling.